src/AppBundle/Command/RemoveBrokenMediaObjectsCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace AppBundle\Command;
+
+use Doctrine\ORM\EntityManager;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+class RemoveBrokenMediaObjectsCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('app:remove-image')
+            ->setDescription('Remove image that not relation to media')
+        ;
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|null|void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $mm = $this
+            ->getContainer()
+            ->get('sonata.media.manager.media')
+        ;
+
+        $em = $this
+            ->getContainer()
+            ->get('doctrine')
+            ->getManager()
+        ;
+
+        $medias = $mm->findAll();
+
+        foreach ($medias as $media) {
+            $context = $media->getContext() == "slider" ? "performance" : $media->getContext();
```

Comment:
And what if I add of edit current contexts? I'll need to change command? It's not good.